arch/arm64/reg.h

```diff
@@ -20,6 +20,8 @@ limitations under the License.
 #define ARCH_SP SP
 #define ARCH_PC PC
 #define ARCH_RETURN_VALUE_REGISTER X0
+#define ARCH_IP_VALUE_REGISTER X0
```

Comment:
These values are specific to the unwinder, so I think they should be defined there and not in reg.h